waspc/src/Wasp/Generator/Shared.hs

```diff
@@ -0,0 +1,29 @@
+module Wasp.Generator.Shared (
```

Comment:
This piece was based heavily on existing Server and WebApp generators `Common` modules, there might be a chance to create some more useful "shared-specific" helper functions.